docs/en/contribution-roadmap/Knowledge.md

```diff
@@ -10,6 +10,17 @@ priority: 10
 
 ## Java
 
+This roadmap assumes you have the basic knowledge about Java:
+* You can read the Alluxio Java source code and get a rough understanding of it.
+* If you find some Java code that you are unfamiliar with,
+  you can search for materials online to understand those Java codes.
+
+Not need to be a Java expert to become an Alluxio contributor, your Java skills
+will improve when you spend time reading the Alluxio source code and contributing to Alluxio.
+
+Note that nobody in the community will or should hand holding you regarding the basic Java knowledge
```

Comment:
hmm this sounds very harsh. i would phrase it more like: you can expect help from the community about Alluxio specific questions but you should be capable of learning about Java on your own since there are lots of online material

but i'll let you phrase that in your own words

Reply:
lol your phrase is better, more polite but same meaning hahaha